packages/generator/src/edmx-parser/common/edmx-types.ts

```diff
@@ -38,6 +42,8 @@ export interface EdmxEntitySetBase extends EdmxNamed {
   'sap:updatable': string;
 }
 
+export type EdmxEntitySetBaseNamespaced = EdmxEntitySetBase & EdmxNamespaced;
```

Comment:
General: We did not use the extension `named` for the object e.g. `EdmxEntitySetBase`. Most of them include the `named` interface and do now the `namespaced` as well. So I would just add then `named` also to the Base sets and not introduce the `Named` name extensions. We could add a note to the competability notes that most of the types include a new property now. Technically this is a breaking change but I  think we can take the risk.

Reply:
Since this is only internal representation of the service metadata.